apps/golden-sample-app/src/app/services/entitlementsTriplets.ts

```diff
@@ -1,6 +1,5 @@
 export const triplets = {
-  canViewTransactions: 'Payments.transactions.view',
-  canViewTransfer: 'Payments.transfer.view',
```

Comment:
Why did you remove this triplet? That was part of the demo about entitlements.

Reply:
`Transactions.canView` is still there, just was renamed, but for the transfer, please, check this comment https://github.com/Backbase/golden-sample-app/pull/31#discussion_r807826609